README.md

```diff
@@ -62,3 +62,11 @@ python3 -m pdfarranger
 ```
 
 For Windows see [Win32.md](Win32.md).
+
+## Translations
+
+*pdfarranger* has been translated into several languages already:
+
+ca@valencia, cs, de, el, es, fr, hr, hu, id, it, ja, nl, pl_PL, pt_BR, tr, zh_CN, zh_TW.po
```

Comment:
Extra .po: zh_TW.*po*

Reply:
Thank you for noticing ... I've corrected it ...